specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/ResourceProvider.json

```diff
@@ -0,0 +1,1675 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CustomHostnameSitesCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get custom hostnames under subscription": {
+            "$ref": "./examples/ListCustomHostNameSites.json"
+          },
+          "Get specific custom hostname under subscription": {
+            "$ref": "./examples/ListCustomSpecificHostNameSites.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/deploymentLocations": {
+      "get": {
+        "summary": "Gets list of available geo regions plus ministamps",
+        "description": "Description for Gets list of available geo regions plus ministamps",
+        "operationId": "GetSubscriptionDeploymentLocations",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DeploymentLocations"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/aseRegions": {
+      "get": {
+        "summary": "Get a list of available ASE regions and its supported Skus.",
+        "description": "Description for get a list of available ASE regions and its supported Skus.",
+        "operationId": "ListAseRegions",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AseRegionCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List aseregions": {
+            "$ref": "./examples/ListAseRegions.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
Why setting it to `null`? The response does have a `nextLink` property.

Reply:
here's my proposed change: https://github.com/Azure/azure-rest-api-specs/pull/28696/commits

Removed nextlinkname from responds body all together

Reply:
responded in another comment